stonesoup/models/base.py

```diff
@@ -224,9 +224,10 @@ def pdf(self, state1, state2, **kwargs):
             The likelihood of ``state1``, given ``state2``
         """
 
+        # Calculate difference before to handle custom types (mean defaults to zero)
+        # This is required as log pdf coverts arrays to floats
         likelihood = multivariate_normal.logpdf(
```

Comment:
I didn't realise this did this in log space by default (indeed, always). This is likely well justified but we should ensure that the users are aware that this is happening by way of a note in the documentation. 

Also, include a switch to allow the the calculation to proceed without logs?

Reply:
The output is wrapped in `Probability` type, so transparently acts as float.

Reply:
What I meant was that there's a potential choice here between `.logpdf()` and `.pdf()` and whereas `.logpdf()` might be a better, indeed obvious, choice, it would be worth documenting that this is what's happening, and perhaps giving users the option of using `.pdf()`.

Reply:
I'm not clear what the advantage would be, as both using `logpdf` wrapped in `Probability` type is effectively same as `pdf` as a float, but using pure float risks issues with values close to zero.

Reply:
I realise that I wasn't very clear about the behaviour. `pdf` is just a wrapper around `logpdf` that calls `exp(log_prob)`. `Probability` type is just a wrapper around a float that is `log_prob`. You use it like a non-logged probability, but if at some point you try to do an operation that doesn't work with a log probability, it'll call `exp(log_prob)`. This gives you advantages of both without having to worry about it.